ansible_collection/hpe/nimble/changelogs/.plugin-cache.yaml

```diff
@@ -0,0 +1,125 @@
+plugins:
```

Comment:
This file should not be part of the repo I think. I see other repos adding .plugin-cache.yaml to .gitignore.